index.bs

```diff
@@ -1884,18 +1942,26 @@ New Charters and Major Changes</h4>
 	(e.g., regarding deliverables or resource allocation)
 	and include rationale for the changes.
 
+	The Call for Review of a new or substantively modified charter
+	<em class="rfc2119">must</em> include a disposition of the comments received during the [=charter refinement=] process,
```

Comment:
```suggestion
	<em class="rfc2119">must</em> include disposition of each comment received during the [=charter refinement=] process,
```

Reply:
I don't mind switching from "the" to "each". However, I think it's best to keep the "a" in front of "disposition", as that better supports the existing understanding and practice that a "disposition of comments" is a document describing how you've handled those comments.

Reply:
```suggestion
	<em class="rfc2119">must</em> include a disposition of comments for those received during the [=charter refinement=] process,
```

Reply:
I think the original wording was clearer here.